src/AddTodoForm.jsx

```diff
@@ -1,17 +1,41 @@
-function AddTodoForm(todoprops) {
+import {useState} from 'react';
+
+function AddTodoForm({ onAddTodo, todoList }) {
+    // New state variable and function to handle title change
+    const [todoTitle, setTodoTitle] = useState("");
+
+    function handleTitleChange(event) {
+        console.log(todoTitle);
+        event.preventDefault();
+        let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    }
+
+    // Function to handle form submission
     const handleAddTodo = (event) => {
         event.preventDefault();
```

Comment:
you need preventDefault here because if you'll not use it the form will be reload and nothing will happen